src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        return List.of(new Bulldozer(),
+                new Bulldozer(), new Bulldozer());

```

Comment:
new line for each entity, and use parameterized constructors to init fields
```suggestion
        return List.of(
                new Bulldozer(),
                new Bulldozer(),
                new Bulldozer()
         );
```